examples/scripts/ct_astra_weighted_tv_admm.py

```diff
@@ -134,7 +134,7 @@ def postprocess(x):
 
 where
 
-  $$W = \mathrm{diag}\left\{\exp( \sqrt{\mathbf{y}}) \right\}.$$
+  $$W = \mathrm{diag}\left\{\exp( \sqrt{\mathbf{y}}) \right\} \;.$$
```

Comment:
Please double check correctness of the formula.

Where does the sqrt come from?
Is there a negative sign missing?

Reply:
We'll have to ask @lukepfister or @Michael-T-McCann to check the equation. It's quite possible that there is a spurious sqrt left over from an earlier change to how we define the weight matrix.

Reply:
See #153.